src/components/Swap/Summary/index.tsx

```diff
@@ -43,7 +43,7 @@ const Body = styled(Column)`
 function Subhead({ impact, slippage }: { impact?: PriceImpact; slippage: Slippage }) {
   return (
     <Row gap={0.5}>
-      {impact?.warning || slippage.warning ? (
+      {Boolean(impact?.warning || slippage.warning) ? (
```

Comment:
This would be more readable as a separate variable:

    const hasWarning = Boolean(impact?.warning || slippage.warning)